beserial/beserial_derive/src/lib.rs

```diff
@@ -213,47 +450,114 @@ fn impl_deserialize(ast: &syn::DeriveInput) -> TokenStream {
         Data::Enum(ref data_enum) => {
             let (enum_type, uvar) = parse_enum_attribs(ast);
 
-            let ty= if uvar {
-                enum_type.unwrap_or_else(|| Ident::new("u64", syn::export::Span::call_site()))
+            let ty = if uvar {
+                enum_type.unwrap_or_else(|| Ident::new("u64", Span::call_site()))
             } else {
-                enum_type.unwrap_or_else(||panic!("Deserialize can not be derived for enum {} without repr(u*) or repr(i*)", name))
+                enum_type.unwrap_or_else(|| {
+                    panic!(
+                        "Serialize can not be derived for enum {} without repr(u*) or repr(i*)",
```

Comment:
Shouldn't this still be "Deserialize" instead of "Serialize"?

Reply:
Copy & paste error :P